doc/2/framework/classes/data-validation/validate/index.md

```diff
@@ -0,0 +1,34 @@
+---
+code: true
+type: page
+title: validate
+description: DataValidation class validate() method
+---
+
+# validate
+
+Validates the content of a request body (mutates the request).
+
+### Arguments
+
+```js
+validate(request: Request, verbose: boolean): Promise<any>;
+```
+
+<br/>
+
+| Arguments | Type                                                           | Description                                                                                         |
+| --------- | -------------------------------------------------------------- | --------------------------------------------------------------------------------------------------- |
+| `request` | [`Request`](/core/2/framework/classes/request) | Request object with a non-empty body content                                                        |
+| `verbose` | <pre>boolean</pre>                                             | If true, returns an exhaustive validation report, instead of failing at the first error encountered |
```

Comment:
```suggestion
| `verbose` | <pre>boolean</pre>                                             | If true, returns an exhaustive validation report, instead of failing at the first encountered error |
```